backend/apps/candidates/models.py

```diff
@@ -37,7 +43,17 @@ class GenderChoices(models.TextChoices):
         MALE = "M", _("Male")
         FEMALE = "F", _("Female")
         OTHER = "O", _("Other")
-    
+
+    class EnglishLevelChoices(models.TextChoices):
+        """This class provides enum for levels of English."""
+        UNKNOWN = "A0", _("Unknown")
+        BEGINNER = "A1", _("Beginner")
+        ELEMENTARY = "A2", _("Elementary")
```

Comment:
A0 its beginner, A1 its elementary, A2 its pre-intermediate

Reply:
According to CEFR there is no such thing as A0 =) I made it up, just out of blue. I used the official table of English levels, https://www.coe.int/en/web/common-european-framework-reference-languages/table-1-cefr-3.3-common-reference-levels-global-scale